### ISSUE TITLE
Pi3 Gamestarter not loading retroarch

### ISSUE THREAD

**Issue by envirotech** [zach-morris/plugin.program.iarl#53]
Been using this for a while.  Recently started a new build, using the newest version of Gamestarter, and now I can't seem to get games to launch from IARL.  Tried setting the retroarch system folder path to
"/storage/.kodi/userdata/addon_data/game.retroarch/" and
 "/storage/.kodi/addons/game.retroarch/"
but neither make any difference.  

Is this maybe because of the change in way the latest Gamestarter installs?

**Comment by zach-morris** [comment]
Yeah, it looks like the newest version of Gamestarter changed a lot.  I'll get updated commands into the next update

**Comment by zach-morris** [comment]
If you have the time, can you confirm this works (with manual entry of the command in IARL):
```
/storage/.kodi/addons/game.retroarch/addon.sh CORE_NAME "%ROM_PATH%"
```
Where core name is the name of the core you want to use, ex: nestopia, genesis_plus_gx, etc

**Comment by envirotech** [comment]
Not sure how to go about that. But I did notice that EmulationStation is configured to:
/storage/.kodi/addons/game.retroarch/addon.start CORE_NAME %ROM% ES

If that helps?

**Comment by zach-morris** [comment]
You can manually enter any command into IARL, it should be the bottom option when you select Update External Command:
https://github.com/zach-morris/plugin.program.iarl/wiki/5.--FAQ#3--i-want-to-use-emulator-xxx-instead

**Comment by envirotech** [comment]
I cant find that.  Is it an option within the Kodi addon settings dialog? Not seeing anything about External Launch Commands. I have my system type set as "RPi Gamestarter Addon", and ran the setup wizard as such.

Pi3 LibreElec 8.1.2

**Comment by envirotech** [comment]
Oh now I get it! 
Tested and yes that works.

**Comment by zach-morris** [comment]
Good deal, thanks for checking

**Comment by rafaelhaertel** [comment]
I have the same problem, when I typed the command you suggested, it seems like it’s gonna load the rom but, instead it just restarts kodi. Using rpi3 with libreelec 8.0.2 here.

**Comment by zach-morris** [comment]
Can you provide a log and the command you typed in?  I'm guessing either:
The core you wanted to launch is not installed or
There's a typo in the command

The commands are in the git now so you can try and copy the command from [here](https://github.com/zach-morris/plugin.program.iarl/blob/master/resources/data/external_command_database.xml#L3609).  An update to the addon should be published soon.

**Comment by rafaelhaertel** [comment]
Hi there, turns out it only worked by sshing the rpi and editing the xml. When i try to input through the app itself, it doesn't work. Dunno if it because i'm doing it via kodi remote app and pasting. But it finally worked. Thank you! Looking forward for the update.

Oh and i dunno if its a server issue, but the PS1 isos are not downloading.

**Comment by zach-morris** [comment]
Fixed in version 1.6.6
